fragments/master.sh

```diff
@@ -19,16 +19,15 @@
 
 set -e
 
+source /etc/sysconfig/heat-params
```

Comment:
This is not so nice. We create a dependency to the environment. In other words /etc/sysconfig/heat-params must exist. I would like to keep master.sh independent of platform. Can you change somehow?